src/applications/vaos/tests/components/cancel/CancelAppointmentModal.unit.spec.jsx

```diff
@@ -73,12 +73,17 @@ describe('VAOS <CancelAppointmentModal>', () => {
       <CancelAppointmentModal
         showCancelModal
         appointmentToCancel={{
-          appointmentType: APPOINTMENT_TYPES.vaAppointment,
-          clinicId: '123',
-          clinicName: 'Testing',
-          facilityId: '123',
+          participant: [
+            {
+              actor: {
+                reference: 'HealthcareService/var983_455',
+                display: 'CHY OPT VAR1',
+              },
+            },
+          ],
+          vaos: { appointmentType: APPOINTMENT_TYPES.vaAppointment },
         }}
-        cernerFacilities={['123']}
+        cernerFacilities={['var983']}
```

Comment:
I don't believe this `cernerFacilities` prop will have the `var` prefix, since it's coming from the profile.